config/base/config.yaml

```diff
@@ -33,22 +33,18 @@ hydra:
           - command
   sweep:
     dir: experiments/multirun  # Directory for sweep runs
-    subdir: ${hydra:runtime.choices.experiment}/sweep/${oc.env:WANDB_SWEEP_ID,local}${run_name}  # Individual sweep run subdirectory
+    subdir: ${hydra:runtime.choices.experiment}/sweep/${oc.env:WANDB_SWEEP_ID,local}/${run_name}  # Individual sweep run subdirectory
 
 # System configuration
 system:
   device: cuda  # We use cuda by default
   run_dir: ${hydra:runtime.output_dir}  # Access run directory at runtime
   data_dir: data  # Root directory in run_dir where all run data is saved
-  checkpoint_step: 5  # Save checkpoints every 5 steps
-  max_checkpoints: 5  # Maximum number of checkpoints to keep
   checkpoint_dir: ${system.data_dir}/checkpoints  # Directory for checkpoints
   plot_dir: ${system.data_dir}/plots  # Directory for plots
   checkpoint_plot_dir: ${system.plot_dir}/checkpoints  # Directory for checkpoint plots
   wandb_preempt: False  # Whether to enable Weights & Biases preemption
```

Comment:
wandb settings stuff is currently spread across 'system' and 'simulation'

Reply:
I've tried reorganizing the configs a bit again. Let me know if you think it can be further improved.